src/components/Menu.jsx

```diff
@@ -1,17 +1,95 @@
 import React from 'react'
 import { IonSegment, IonSegmentButton } from '@ionic/react'
+import { Link } from 'react-router-dom'
+import { connect } from 'react-redux'
+import { CATEGORY_SELECTION } from '../state/actions/actionTypes'
 
-const Menu = () => {
+const Menu = props => {
+  const handleItemClick = event => {
+    props.dispatch({
+      type: CATEGORY_SELECTION,
+      payload: {
+        categoryName: event.target.id,
+        activeItem: event.target.active
+      }
+    })
+  }
   return (
-    <IonSegment>
-      <IonSegmentButton>All News</IonSegmentButton>
-      <IonSegmentButton>Latest News</IonSegmentButton>
-      <IonSegmentButton>Culture</IonSegmentButton>
-      <IonSegmentButton>Tech</IonSegmentButton>
-      <IonSegmentButton>Food</IonSegmentButton>
-      <IonSegmentButton>Sports</IonSegmentButton>
+    <IonSegment id='article-category' scrollable>
+      <IonSegmentButton
+        name='All News'
+        id=''
+        color='yellow'
+        as={Link}
+        to={{ pathname: '/' }}
+        onClick={handleItemClick}
+        active={props.activeItem === 'all'}
+      >
```

Comment:
I think we can remove the color= and the active= and also the activeItem from other places in the code base. I am 90% sure these were just for color with semantic but test it out to be sure everything works as expected without them.

Reply:
We discussed and this will be done as a chore later if there is time as it is not a blocker on functionality but would require a larger refactoring effort.